src/Executor/Executor.php

```diff
@@ -42,45 +51,41 @@ public function __construct(string $endpoint)
      * @param string $deploymentId
      * @param string $projectId
      * @param string $source
-     * @param string $runtime
-     * @param string $baseImage
+     * @param string $image
      * @param bool $remove
      * @param string $entrypoint
      * @param string $workdir
-     * @param string $destinaction
-     * @param string $network
-     * @param array $vars
+     * @param string $destination
+     * @param array $variables
      * @param array $commands
      */
     public function createRuntime(
         string $deploymentId,
         string $projectId,
         string $source,
-        string $runtime,
-        string $baseImage,
+        string $image,
         bool $remove = false,
         string $entrypoint = '',
         string $workdir = '',
         string $destination = '',
-        array $vars = [],
+        array $variables = [],
         array $commands = []
     ) {
+        $runtimeId = "$projectId-$deploymentId";
         $route = "/runtimes";
-        $headers = [
-            'content-type' => 'application/json',
-            'x-appwrite-executor-key' => App::getEnv('_APP_EXECUTOR_SECRET', '')
-        ];
+        $headers = array_merge($this->headers, [ 'x-opr-runtime-id' => $runtimeId ]);
```

Comment:
You dont need to array_merge here.
Its already done here
https://github.com/appwrite/appwrite/blob/192cf2e20d2f5cf0f24e4778e8adedbf9287aa84/src/Executor/Executor.php#L170

Reply:
Ooooh now I get it. Updated.